### ISSUE TITLE
Create an endpoint for the contact form on the McKenzie site

### ISSUE THREAD

**Issue by RyanPnayR** [RyanPnayR/issue-migration#15]
In GitLab by @randseay on Sep 4, 2017, 14:41

It makes sense for all of the fields to be required. If I could get a response with a success/error indicator, that would be great. I would like to `post` using `fetch()`, which would send this `json` response:

```json
{
    email: string;
    name: string;
    phone: string;
    text: string;
}
```

**Comment by RyanPnayR** [comment]
In GitLab by @nZac on Oct 4, 2017, 09:27

closed via commit 653e75455eac2998f5108eaae3ab130a165ef462

**Comment by RyanPnayR** [comment]
In GitLab by @nZac on Oct 4, 2017, 09:27

closed via commit 653e75455eac2998f5108eaae3ab130a165ef462

**Comment by RyanPnayR** [comment]
In GitLab by @nZac on Oct 4, 2017, 09:27

closed via commit 653e75455eac2998f5108eaae3ab130a165ef462
